<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="btClose">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>440</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Close</string>
    </property>
   </widget>
   <widget class="QTabWidget" name="Tabs">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>800</width>
      <height>480</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>2</number>
    </property>
    <widget class="QWidget" name="tabGen">
     <attribute name="title">
      <string>General</string>
     </attribute>
    </widget>
    <widget class="QWidget" name="tabCons">
     <attribute name="title">
      <string>Consulta</string>
     </attribute>
    </widget>
    <widget class="QWidget" name="tabAju">
     <attribute name="title">
      <string>Ajustes</string>
     </attribute>
     <widget class="QLabel" name="lbIP">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>78</y>
        <width>111</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <widget class="QLabel" name="lbHTTP">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>60</y>
        <width>121</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <widget class="QTableWidget" name="tabla">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>240</y>
        <width>221</width>
        <height>192</height>
       </rect>
      </property>
      <column>
       <property name="text">
        <string>IP</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Name</string>
       </property>
      </column>
     </widget>
     <widget class="QPushButton" name="testMaxi">
      <property name="geometry">
       <rect>
        <x>670</x>
        <y>70</y>
        <width>80</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>PushButton</string>
      </property>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>420</x>
        <y>50</y>
        <width>221</width>
        <height>21</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Direcci√≥n IP</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Nombre</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="horizontalLayoutWidget">
      <property name="geometry">
       <rect>
        <x>420</x>
        <y>70</y>
        <width>221</width>
        <height>31</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout"/>
     </widget>
    </widget>
   </widget>
   <zorder>Tabs</zorder>
   <zorder>btClose</zorder>
  </widget>
 </widget>
 <tabstops>
  <tabstop>Tabs</tabstop>
  <tabstop>btClose</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
